packages/terra-data-grid/src/DataGrid.module.scss

```diff
@@ -13,6 +13,8 @@
     // Style for the grid itself
     border-collapse: separate;
     border-spacing: 0;
+    // Allow for resizing of last grid column
+    margin-right: 30px;
```

Comment:
does this holds good for all the viewports

Reply:
This provides extra space so that the final column can be resized.  Do you have an example in mind where maybe more room is required?  We also did not want to take too much room from the table itself.

Reply:
Since the value has provided with `px` this would remain same on all the viewports. There might be few scenarios where data-grid might have minimum columns or it might get required space to get displayed without scroll in such scenarios due to this margin we might notice scroll because of the additional space added by margin OR Extra white space only on right-end part of grid. Curious to know if this change was suggested / approved by UX team.

<img width="1137" alt="Screenshot 2023-10-19 at 6 36 46 PM" src="https://github.com/cerner/terra-framework/assets/12869809/8335a9b7-6a56-4850-850f-a710c4965f74">

Reply:
I think it's for giving the last resize handle a but more room to be grabbed. Do we still want it? 
cc: @cm9361

Reply:
Oh yeah is this the little bit of extra space that the handle needs to be able to expand? If so, is that the smallest it can be?

Reply:
Yes, this is the extra space that the handle needs to expand for the last row.  I tried to pick a value that would be comfortable for users.  I am sure we can select a smaller value and it still work.

Reply:
Reduced to 15px here: 17fa632348ba196e03112e7502690d002bbc1c68